dds/DCPS/RTPS/RtpsDiscovery.cpp

```diff
@@ -89,6 +89,8 @@ RtpsDiscoveryConfig::RtpsDiscoveryConfig()
   , secure_participant_user_data_(false)
   , max_type_lookup_service_reply_period_(5, 0)
   , use_xtypes_(XTYPES_MINIMAL)
+  , type_lookup_service_default_enum_extensibility_(DDS::APPENDABLE)
+  , type_lookup_service_default_bitmask_extensibility_(DDS::APPENDABLE)
```

Comment:
Is it necessary to have a `bitmask` option for this?  There aren't (yet) any opendds_idl-generated TypeObjects for bitmasks

Reply:
I would say yes and more.  In general, we should be checking received type objects to ensure that _all_ flags and accouterments are set to valid values.